frontend/src/assets/css/variables.scss

```
@@ -0,0 +1,7 @@
+:root {
+  --white-pale: #f8f9fa;
+  --black-shadow: 0 2px 4px #0000001a;
+  --lightgray: #e2e3e5;
+  --black: #000000;
+  --medium-gray: #7c7f89;
```

We have a convention that we name color variables like gray-100, where number is intensity of the color (we chose ourselves), except white and black colors
```suggestion
  --black: #000000;
  --gray-100: #e2e3e5;
  --gray-200: #7c7f89;
```